### PR TITLE
Add Enphase_envoy prerequisites

### LINKED ISSUE #34529: HomeAssistant > Enphase > Configuration Prerequisite - Enphase Devices should be on your network.
### Feedback

I read the configuration details and it says "Host' is required, but I haven't talked with my installers to configure the local wifi configuration (it is still using the installer's cellular configuration).  Adding a simple statement as to this prerequisite would be helpful so that users who have not yet put this onto their network know they have a prerequisite to satisfy.

### URL

https://www.home-assistant.io/integrations/enphase_envoy/

### Version

2023.11.3

### Additional information

(taking another swing at moving to Home Assistant, so my versions are older).

### PR DESCRIPTION
**Pull request by catsmanac**
## Proposed change

Describe more clearly what the prerequisites are for using Enphase envoy integration with Home Assistant.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #34529

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Enhanced the Enphase Envoy integration documentation with a new "Prerequisites" section, detailing necessary configurations and requirements for successful integration.

### CODE REVIEW

File: source/_integrations/enphase_envoy.markdown [hunk 1]
```diff
@@ -23,7 +23,16 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-An integration for the [Enphase Envoy](https://enphase.com/en-us/products-and-services/envoy-and-combiner) solar energy gateway. This integration works with older models that only have production metrics (ie. Envoy-C) and newer models that offer both production and consumption metrics (ie. Envoy-S). Firmware version 3.9 or newer is required.
+An integration for the [Enphase Envoy](https://enphase.com/en-us/products-and-services/envoy-and-combiner) solar energy gateway. This integration works with older models that only have production metrics (ie. Envoy-C) and newer models that offer both production and consumption metrics (ie. Envoy-S).
+
+## Pre-requisites
```
Comment (c0ffeeca7) [inline source/_integrations/enphase_envoy.markdown]:
```suggestion
## Prerequisites
```

File: source/_integrations/enphase_envoy.markdown [hunk 2]
```diff
@@ -23,7 +23,16 @@ ha_platforms:
...
+
+- The Envoy must be configured and commissioned.
+- The Envoy must be on your local network with IPV4 connectivity from Home Assistant.
+- Envoy firmware version 3.9 or newer, use a custom integration with older firmware ([1](https://github.com/catsmanac/ha_enphase_envoy_with_legacy),[2](https://github.com/briancmpbll/home_assistant_custom_envoy)).
```
Comment (c0ffeeca7) [inline source/_integrations/enphase_envoy.markdown]:
```suggestion
```
We don't usually allow linking to custom integration repos from the documentation

Comment (catsmanac) [reply source/_integrations/enphase_envoy.markdown]: Ah, I see. I'll remove that part from the sentence. First part before the , should be kept. Thanks for reviewing

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @catsmanac :+1:

**Pull request merged**